Sources/Instructions/Extra/Default Views/CoachMarkArrowDefaultView.swift

```diff
@@ -25,34 +25,88 @@ import UIKit
 // MARK: - Default Class
 /// A concrete implementation of the coach mark arrow view and the
 /// default one provided by the library.
-public class CoachMarkArrowDefaultView: UIImageView, CoachMarkArrowView {
+public class CoachMarkArrowDefaultView: UIView, CoachMarkArrowView {
```

Comment:
The arrow should remain an `UIImageView` for now. See the main comment for details.